docs/project-setup/filesystem-sync.md

```diff
@@ -48,12 +48,11 @@ The file synchronization is managed by the rig cli via `rig project sync:start`
 
 The name of the sync volume can be set in a variety of ways. It is determined using the following precedence:
 
-1. Argument provided to the `project sync:start` command
-2. Specified in the `sync` -> `volume-name` property of your [Outrigger Project Configuration](./project-configuration.md)
-3. In your `docker-compose.yml` file, specified as an `external` volume with a name of the pattern `*-sync`
-4. If not specified anywhere else, it will use the name of the current project folder with `-sync` appended
+1. Specified in the `sync` -> `volume-name` property of your [Outrigger Project Configuration](./project-configuration.md)
+2. In your `docker-compose.yml` file, specified as an `external` volume with a name of the pattern `*-sync`
+3. If not specified anywhere else, it will use the name of the current project folder with `-sync` appended
```

Comment:
If they use the `--dir` option it will use the current project folder of that location.